### DISCUSSION TITLE
[Bug]: Disabled cameras appear in previews

### DISCUSSION THREAD

**Discussion by sshaikh** [blakeblackshear/frigate#13100] [Report a Bug] [answered]
### Describe the problem you are having

In the list of cameras to the right of the main one, I see a camera that has been disabled in the config. It does not appear elsewhere.

### Steps to reproduce

Disable camera in config. It still appears in the mini screens.

### Version

0.14.1-4dce8ff

### Frigate config file

```yaml
NA
```

### Relevant log output

```shell
NA
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

NA

**Comment by NickM-27** [comment, marked as answer]
this is not a bug. Just because a camera is disabled right now does not mean that it has always been disabled, and does not mean it was disabled at the time that is currently being viewed

**Reply by sshaikh** [reply]
Ok this makes sense - this camera has been disabled for over a year (it was used for detect testing) so I presume it will appear even if there is no footage or detect data for the date being reviewed?

If so, is there a way to keep the config but disable it from the UI? Or is it just a matter of commenting it out (vs simply disabling it?)

**Reply by NickM-27** [reply]
if you have not enabled it in that long of time, yes I would just comment it out
